packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -338,84 +91,58 @@ export default function Sidebar({
             : 'laptop:top-14 laptop:h-[calc(100vh-theme(space.14))]',
         )}
       >
-        {sidebarRendered && (
-          <MenuIcon
-            sidebarExpanded={sidebarExpanded}
-            toggleSidebarExpanded={trackAndToggleSidebarExpanded}
-          />
-        )}
+        {sidebarRendered && <MobileMenuIcon />}
         <SidebarScrollWrapper>
           <Nav>
             <SidebarUserButton sidebarRendered={sidebarRendered} />
             {!alerts?.filter && (
               <MyFeedButton
                 sidebarRendered={sidebarRendered}
                 sidebarExpanded={sidebarExpanded}
-                item={myFeedMenuItem}
-                isActive={activePage === myFeedMenuItem.path}
-                useNavButtonsNotLinks={useNavButtonsNotLinks}
+                activePage={activePage}
+                isButton={isNavButtons}
+                alerts={alerts}
+                onNavTabClick={onNavTabClick}
               />
             )}
-            <RenderSection
+            <DiscoverSection
               {...defaultRenderSectionProps}
-              title="Discover"
-              items={discoverMenuItems}
-              useNavButtonsNotLinks={useNavButtonsNotLinks}
+              popularFeedCopy={popularFeedCopy}
+              onNavTabClick={onNavTabClick}
+              enableSearch={enableSearch}
+              isItemsButton={isNavButtons}
             />
-            {!!contributeMenuItems.length && (
-              <RenderSection
-                title="Contribute"
-                items={contributeMenuItems}
-                {...defaultRenderSectionProps}
-                useNavButtonsNotLinks={false}
-              />
-            )}
-            <RenderSection
+            <ContributeSection
               {...defaultRenderSectionProps}
-              title="Manage"
-              items={manageMenuItems}
-              useNavButtonsNotLinks={false}
+              canSubmitArticle={canSubmitArticle}
+              submitArticleOn={submitArticleOn}
+              submitArticleSidebarButton={submitArticleSidebarButton}
+              submitArticleModalButton={submitArticleModalButton}
+              onTrackEvent={trackEvent}
```

Comment:
Sorry for messing you around, but this actually makes more sense closer to implementation as it's not re-used.

I think we can safely say tracking can happen inside component, all other should rather be drill-down?

Reply:
No worries, I shall make that happen! 🚢